### ISSUE TITLE
Clarifying menus in general settings vs scene menu

### ISSUE THREAD

**Issue by kingborehaha** [Grimrukh/soulstruct-blender#2]
> In addition, each file type has "quick" Import/Export operators that attempt to make it very easy to import and export supported files from a supported game installation, using the General Settings panel in the 3D View or the Scene Properties tab.

This line made me think the two menus had quick import/export functionality, and I got a little confused because I found the scene menu, but didn't find the general settings menu (I had to ask for help to find it too 😔). I think the wording can be clarified if you don't want to change which options are present in which menus.

Also as a non-blender user, I think this would be handy to include in the readme:
![image](https://github.com/Grimrukh/soulstruct-blender/assets/55667610/8400a287-048e-46fe-8bc0-62dbdcec0866)

**Comment by Grimrukh** [comment]
Good idea, will definitely add some better "Blender 101" notes and images to the README soon!
